### ISSUE TITLE
React Router + Redux + SSR guide

### ISSUE THREAD

**Issue by samouss** [algolia/react-instantsearch#1211]

**Comment by ReneF1** [comment]
Greetings! @samouss any progress on this issue ? I'm stuck with implementing algolia in my nextjs project because i am relying on redux also.

The problem seems to be this part of the code:

```
    static async getInitialProps(ctx) {
        const {store, asPath} = ctx
        const searchState = asPath.includes('?')
            ? qs.parse(asPath.substring(asPath.indexOf('?') + 1))
            : {};
        const resultsState = await findResultsState(SearchComponent, {searchState});
        store.dispatch(getIsoTime())
        const filter = store.getState().filter;
        const isoTime = store.getState().isoTime;
        return {filter, isoTime, resultsState, searchState}
    }
```
Invariant Violation: Could not find “store” in either the context or props of “Connect(SearchComponent)”

How can i make the store accessible server side with findResultsState(...) ?

**Comment by samouss** [comment]
No progress on this issue at the moment. The problem seems to come from the `SearchComponent` that want to access the Redux `store`. To have access to it the components must be scoped under the Redux `Provider`. The component provided to `findResultsState` must be self-contained. Here the `SearchComponent` is not because the `store` is not accessible. You can [wrap the component](https://github.com/samouss/react_instant_question/blob/3f17182739305ded28cba3748bbef3fee3fe7249/src/server.js#L19-L23) with a Redux provider.  If the issue persist please provide a live example, it would help to better understand the issue. Here is [a template](https://codesandbox.io/s/github/algolia/create-instantsearch-app/tree/templates/react-instantsearch) to avoid the boilerplate part.

**Comment by alexvandervegt** [comment]
Any update on this?

**Comment by Haroenv** [comment]
No, sorry @alexvandervegt, is there something we can particularly help with? 
Can you create what you have so far in a sandbox environment? We have a template available for that purpose [here](https://codesandbox.io/s/github/algolia/create-instantsearch-app/tree/templates/react-instantsearch). Thanks!

**Comment by sarahdayan** [comment]
Now that React InstantSearch Hooks is preferred over React InstantSearch, and will soon replace it, we'll much likely not document this.

If there's interest for missing guides for React InstantSearch Hooks, don't hesitate to open a new issue.
